modules/ExpensifyNitroUtils/.gitignore

```diff
@@ -0,0 +1,70 @@
+# OSX
```

Comment:
NAB but since this isn't a submodule we could just use the `.gitignore` at the root, right?

Reply:
not important to change this, we can keep the 2nd `.gitignore`

Reply:
Indeed, it's not a git submodule, and we can use .gitignore from the root. The reason I left it separate was just to keep these specific patterns isolated. However, since we expect to have a new Nitro module in the future I'm moving it to the root